_posts/2021-10-01-how-to-pre-assignments.md

```diff
@@ -0,0 +1,67 @@
+
+## README
+If your project is published in Github, the first thing we will see of your pre-assignment is README. With good README we get a clear picture of your project even without looking at the code or running it. 
+
+A good README includes following things:
+- Prerequisites. Should the reviewer install something on their computer before they can compile and run the project? Does the project only work on Windows or on Linux? List clearly all steps that need to be done before trying to run the project. If versions are important, remember to mention those as well.
+- Configurations. Do you have to configure for example database connections locally? Provide clear instructions on what needs to be changed and where. In case you have an .env file which you, of course, should not add to GitHub, you can send that file to the reviewers by other means.
+- How to run the project? Do you have to install some packages or compile the code? If you have separate services for example for backend and frontend, remember to write instructions for all needed services.
+- Tests. If your project has tests, include instructions on how to run them.
+- Description of the project. What is the purpose of the project and what features it has?
+- Technology choices. List chosen technologies. It's also nice to know why you chose those technologies.
+- TODO. If there are some things missing or not working, you can list them in README.
+
+## Git History
+Your git history is also something that we will often look into before digging deeper into your code. From your git history we can get a glimpse of your development process: where you started, in which order you did things, which parts were refactored during the process, etc. Knowing how to use version control (nowadays analogous to git) is one of the most essential skills of a developer. To make your project look more professional you should pay some attention to how your git history looks. 
+
+During the Dev Academy recruitment process we saw many git histories that had basically two commits. One containing all the application code and the second updating readme. Please try to have more commits than that. A good rule of thumb is to have one commit per one major change in your code. A good commit message describes the changes your commit makes to the repository. Try to be concise but yet descriptive, not easy, we know :) Try to think what is the one main thing you did in this commit, if there are several, then maybe that is an indication that you should have several commits instead of one.
+
+Some people like to keep they git history extra neat by amending and squashing their commits. In a recruitment pre-assignment it is usually enough to just commit frequently enough and in reasonably sized chunks.
```

Comment:
some people like to keep _their_ git history...

in a recruitment pre-assignment, it is usually....